zerver/views/__init__.py

```diff
@@ -874,6 +874,7 @@ def home(request):
         fullname              = user_profile.full_name,
         email                 = user_profile.email,
         domain                = user_profile.realm.domain,
+        domains               = list_of_domains_for_realm(user_profile.realm),
```

Comment:
yeah. I was trying to optimize for the common case where there is exactly 1 domain. Though now that subdomains is merged there a second common case of 0 domains, which I still need to take care of.